ogticrd/covid-safe-paths
# Epidemiological response screen

#### Description:
Create a screen to give support to COVID-19 Positive users, where they can see their epidemiological status and some mental health advices. Also changed the flow of the Report after the user insert their ID's so we can check if they're positive from the begining.



#### Linked issues:
[Ticket 88](https://trello.com/c/19ndqu9f/88-screen-for-epidemiological-response)


#### Screenshots:
![Simulator Screen Shot - iPhone 11 - 2020-06-03 at 16 42 45](https://user-images.githubusercontent.com/56127045/83687173-61689c80-a5b9-11ea-85cb-042f0c023e85.png)
![Simulator Screen Shot - iPhone 11 - 2020-06-03 at 16 42 50](https://user-images.githubusercontent.com/56127045/83687175-62013300-a5b9-11ea-9886-451421e1653d.png)
![Simulator Screen Shot - iPhone 11 - 2020-06-03 at 16 42 54](https://user-images.githubusercontent.com/56127045/83687176-6299c980-a5b9-11ea-8896-ba71097dd8c2.png)
![Simulator Screen Shot - iPhone 11 - 2020-06-03 at 16 43 25](https://user-images.githubusercontent.com/56127045/83687178-63326000-a5b9-11ea-8b36-4c0a1d155094.png)



#### How to test:
You can open this screen in Settings > Epidemiological report > completing your ID info (if you are positive you'll see the screens listed in the screenshots, if not, you'll have to fill up an Report. You can only see the screen if you're positive)
